server/store.go

```diff
@@ -140,25 +140,25 @@ func (p *Plugin) storeMeetingPostID(meetingID int, postID string) *model.AppErro
 	return p.API.KVSetWithExpiry(key, bytes, meetingPostIDTTL)
 }
 
-func (p *Plugin) fetchMeetingPostID(meetingID string) (string, *model.AppError) {
+func (p *Plugin) fetchMeetingPostID(meetingID string) (string, error) {
 	key := fmt.Sprintf("%v%v", postMeetingKey, meetingID)
-	postID, appErr := p.API.KVGet(key)
-	if appErr != nil {
-		p.API.LogDebug("Could not get meeting post from KVStore", "error", appErr.Error())
-		return "", appErr
+	var postID string
+	if err := p.client.KV.Get(key, postID); err != nil {
```

Comment:
I think we should pass in a `*string` right?

```suggestion
	if err := p.client.KV.Get(key, &postID); err != nil {
```